.github/workflows/msys2release.yml

```diff
@@ -0,0 +1,69 @@
+name: release
+
+on:
+  push:
+    tags:
+        - "v*.*.*"    
+  
+jobs:
+  windows-rocksdb-x86_64:    
+    runs-on: windows-latest
+    environment: release    
+    steps:
```

Comment:
https://github.com/crypto-org-chain/cronos/blob/main/.github/workflows/release.yml#L27
you need to add the `needs: ["goreleaser"]` to avoid draft duplicated release.
maybe you also need to move the job to the release.yml to use that.

Reply:
ok, i'll move to release.yml
and add `needs`